jupyter-6.4.0/Dockerfile

```diff
@@ -0,0 +1,113 @@
+# Copyright (c) Jupyter Development Team.
+# Distributed under the terms of the Modified BSD License.
+FROM quay.io/cdis/scipy-notebook:f1a4a37136d8
```

Comment:
why not `notebook-6.4.0` instead of `f1a4a37136d8`, for clarity

Reply:
sure, I will update😊